app/views/posts/show.html.erb

```diff
@@ -18,5 +18,9 @@
 
 </ul>
 
+<div class="button-container">
+<a href= "<%= new_user_post_comment_path(@user_id, @post) %>" class="btn"> Add Comment</a>
+</div>
```

Comment:
- [ Required ] I think it would be nice if you add a mechanism for adding a `like` to a post in your App the same way you've done for a `comment`. This feature is required according to the project guidelines, please add it.

## Project requirements
![Screenshot from 2023-08-26 18-10-48](https://github.com/isume295/Blog-app/assets/16392046/366662b2-b7c0-4869-8b9e-94314b7f05f1)